webpack/components/FormEdit.js

```diff
@@ -170,9 +170,12 @@ class FormEdit extends Component {
       return (event) => {
         let index = parseInt(event.target.getAttribute("data-question"));
         let newState = Object.assign({}, container.state);
-        newState.formQuestions[index].responseSetId = event.target.value;
-        container.setState(newState);
-        this.unsavedState = true;
+        const responseSetId = parseInt(event.target.value);
+        if (! isNaN(responseSetId)) {
```

Comment:
@jmazella fixed this as well a bit differently in another branch so I am just @'ing him so we can take a look and decide how to handle for the rebase today